api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -229,6 +231,41 @@
     public void fireEvent(Object event, Annotation... qualifiers);
 
     /**
+     * Fire an asynchronous event and notify observers.
```

Comment:
I think it can be useful for code outside CDI. For instance user can get beanmanager from JNDI and fire an event asynchronously to trigger code in the bean graph without having to add CDI code to the calling module.